core/src/alpenglow_consensus/voting_loop.rs

```diff
@@ -797,7 +819,7 @@ impl VotingLoop {
         my_pubkey: &Pubkey,
         slot: Slot,
         root_bank_cache: &mut RootBankCache,
-        cert_pool: &mut CertificatePool<LegacyVoteCertificate>,
+        cert_pool: &mut CertificatePool,
         voting_context: &mut VotingContext,
     ) {
```

Comment:
This function should also rebroadcast any observed certificates for slots after the finalized one. Also, the `TODO` comment mentions sending a NotarFallback certificate for the finalized slot (if slow finalized), this should be a Notar certificate instead.